tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -792,7 +790,9 @@ class MklConvOp : public OpKernel {
         // Tensorflow format to MKL format by caching the filter when it is
         // converted for the first time. This cached filter can then be reused
         // in subsequent iterations.
-        if (is_filter_const_) {
+        bool do_cache_filter =
+            src_dims[MklDnnDims::Dim_N] > kSmallBatchSize ? true : false;
```

Comment:
Nit: The expression already returns boolean.
```suggestion
            (src_dims[MklDnnDims::Dim_N] > kSmallBatchSize);
```

Reply:
Modified! Thanks!